RPC.c

```diff
@@ -0,0 +1,62 @@
+/*Lloyd Page*/
```

Comment:
Good job, overall! Your code did give out a warning, which you should always try to avoid. Your implementation was good though, and I appreciated the repeatability feature!